drivers/ethernet/eth_enc424j600.c

```diff
@@ -644,9 +642,9 @@ static int enc424j600_init(struct device *dev)
 		return -EINVAL;
 	}
 
-	if (gpio_pin_enable_callback(context->gpio, config->gpio_pin)) {
-		return -EINVAL;
-	}
+	gpio_pin_interrupt_configure(context->gpio,
+				     config->gpio_pin,
+				     GPIO_INT_EDGE_TO_ACTIVE);
```

Comment:
isn't GPIO_INT_ENABLE missing?

Reply:
GPIO_INT_ENABLE is a part of GPIO_INT_EDGE_TO_ACTIVE [macro](https://github.com/zephyrproject-rtos/zephyr/blob/topic-gpio/include/drivers/gpio.h#L175).

Reply:
ok, the documentation is a bit misleading

Reply:
`GPIO_INT_EDGE_TO_ACTIVE` is a convenience macro targeted at users of the GPIO API. `GPIO_INT_ENABLE` is a macro targeted at the GPIO driver developers. In the doxygen generated documentation only convenience flags will be visible. This is fairly popular convention. Nevertheless, if someone has an idea how this can be improved it's not too late.